arch/x86/guest/vmcall.c

```diff
@@ -106,6 +106,10 @@ int vmcall_vmexit_handler(struct vcpu *vcpu)
 		ret = hcall_set_vm_memmap(vm, param1, param2);
 		break;
 
+	case HC_VM_SET_MEMMAPS:
```

Comment:
are you supporting post-vmluanch EPT mapping configuration update? or runtime EPT mapping update?
For example, if you update the normal world EPT mappings, but in its secure world (trusty), there is no changes in sworld_eptp. Then afterwards, if Trusty attempts to access that memory region (for some reason), then we may have problems. 

however, we're not allowed to do EPT mapping updates for secure world at runtime.